backend/app.py

```diff
@@ -442,7 +437,64 @@ def change_machine():
     return jsonify({"message": "Success!"}), 200
 
 
+@app.route("/trigger_location_differ", methods=["POST"])
+def trigger_location_differ():
+    """
+    Triggers the location differ script.
+    """
+    request_queue.put((run_location_differ, ()))
+    return jsonify({"message": "Success!"}), 200
+
+
+def run_location_differ():
+    with setup_locdiffer_logger():
+        old_json_file = "/root/PennyMe/new_data/old_server_locations.json"
+        new_json_file = "/root/PennyMe/new_data/server_locations.json"
+        new_problems_json_file = "/root/PennyMe/new_data/problems.json"
+        debug_path = "/root/PennyMe/debug_new_data"
```

Comment:
Docstring missing, and shouldn't we set these paths as capital letter variables in the beginning of the file or so?

Reply:
Added the docstring, I'm against uppercasing because this is typically done for global not for local variables